store/src/java/com/zimbra/cs/account/auth/AuthContext.java

```diff
@@ -73,6 +73,26 @@
      */
     public static final String AC_DEVICE_ID = "did";
 
+    private static final Map<String, String> defaultUserAgentMap = new HashMap<String, String>();
```

Comment:
Please change key of Map. It should be Protocol, like this -
private static final Map<Protocol, String> defaultUserAgentMap = new HashMap<Protocol, String>();